src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,45 @@
 package core.basesyntax;
 
-public class WorkWithFile {
-
-    public void getStatistic(String fromFileName, String toFileName) {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
+    public static void getStatistic(String fromFileName, String toFileName) {
```

Comment:
why static? also please create private methods for different parts of logic (reading, calculating, writing) and call them one by one